### PR TITLE
If cargo not installed, install to /usr/local/bin

### PR DESCRIPTION
**Pull request by devanonon**
I guess now with the releases the whole point is it should work even if rust / cargo are not installed. I tested it on another machine without rust and had an issue that `~/.cargo/bin` does not exist. I propose that if that's the case, we install to `/usr/local/bin`.

I also fixed a typo in the final output of installing `forgeup`.

### CODE REVIEW

**Comment by onbjerg** [comment]
We should probably just move `forge` and `cast` to `$FORGEUP_DIR` since users already need to add that to their `PATH` to run `forgeup` in the first place

**Comment by gakonst** [comment]
Should we also rename to `foundryup` while we're at it, given that it installs both cast and forge?

**Comment by onbjerg** [comment]
That would make sense

**Comment by gakonst** [comment]
I'm also +1 on installing to $FOUNDRY_DIR (let's use that as a name for all our installations instead of FORGEUP_DIR, given there may be more things in the future, e.g. caching for RPC?)

**Comment by onbjerg** [comment]
We should probably install `foundryup` in `$FOUNDRY_DIR/bin` in that case to keep it all neat

**Comment by gakonst** [comment]
Yes, exactly. @devanonon otherwise agreed with your changes, could you add the above in the PR? (doing everything under `FOUNDRY_DIR/bin`)

**Comment by devanonon** [comment]
Sure, thanks for the super quick feedback guys! Will come back with some more commits addressing these points shortly.

**Comment by devanonon** [comment]
Damn, renaming the dir and making significant changes kind of screwed up the diff. This might help with reviewing: https://www.diffchecker.com/AKCKoXmQ

File: README.md [hunk 1]
```diff
@@ -21,12 +21,16 @@ Foundry consists of:
 ## Installation 
 
 ```
-curl https://raw.githubusercontent.com/gakonst/foundry/master/forgeup/install | bash && forgeup
+curl https://raw.githubusercontent.com/gakonst/foundry/master/foundryup/install | bash
+```
+in a new terminal session or after reloading your PATH:
+```
+foudryup
```
Comment (onbjerg) [inline README.md:28]: missed an `n` here across the board in the file :smile:

File: README.md [hunk 2]
```diff
@@ -21,12 +21,16 @@ Foundry consists of:
...
 ```
 
-Just get forgeup, the Foundry toolchain installer, and use it to install the latest `forge` and `cast` binaries.
+Just get foudryup, the Foundry toolchain installer, and use it to install the latest `forge` and `cast` binaries.
```
Comment (gakonst) [inline README.md:31]: typo `foundryup`

**Comment by gakonst** [comment]
Yeah the diff is fine, lgtm too

**Comment by devanonon** [comment]
Oops, bash is easier than english. 😄 Fixed typos, thanks for spotting!

**Review by gakonst — APPROVED** [review]

**Pull request merged**